features/async-generators.yml

```diff
@@ -0,0 +1,17 @@
+name: Async generators
+description: Async generator functions (`async function*`) are asynchronous and always yield `Promise` objects.
+group: javascript
```

Comment:
Would it make sense to have a `generators` group?

Reply:
Let me add that as a follow up when this PR and https://github.com/web-platform-dx/web-features/pull/1664 have been merged

Reply:
These are both merged now. 👍